freightliner_cascadia_2012_dot_10002/bridge.yml

```diff
@@ -9,6 +9,14 @@ topics:
     topic: /hardware_interface/lidar/points_raw
     type: sensor_msgs/msg/PointCloud2
     queue_size: 1
+  -
+    topic: /hardware_interface/velodyne_1/lidar/points_raw
+    type: sensor_msgs/msg/PointCloud2
+    queue_size: 3
+  -
```

Comment:
Can we add /system_alert -> carma_msgs/msg/SystemAlert to the bridge.yml files as well?

Reply:
Huh, I could have sworn that was already there, but i don't see it. I'll add it